plugins/synaptics-mst/fu-synaptics-mst-device.c

```diff
@@ -860,6 +860,15 @@ fu_synaptics_mst_device_update_cayenne_firmware(FuSynapticsMstDevice *self,
 	guint32 offset = 0;
 	guint32 write_loops = 0;
 
+	/* sanity check */
+	if (payload_len < 0x50000) {
```

Comment:
3 places using a hardcoded `0x50000` tells me this should be a `#define`

Reply:
Fair point, updated.